src/usb_serial.rs

```diff
@@ -0,0 +1,29 @@
+#![allow(non_camel_case_types)]
```

Comment:
As per above, maybe switch the terminology from "usb_serial" to just "jtag" everywhere?